0-bootstrap/cb.tf

```diff
@@ -134,6 +137,15 @@ module "tf_source" {
   depends_on = [module.seed_bootstrap]
 }
 
+resource "google_project_service_identity" "workflows_identity" {
```

Comment:
Curious why this is requried?

Reply:
this one solved a frequent build error that this service account did not exist when building the Terraform image used in the CB builds.

https://console.cloud.google.com/cloud-build/builds/6cc5e896-452e-4f32-a6d3-6ff4ffa25138?project=cloud-foundation-cicd

```
Step #3 - "converge-bootstrap":   with module.tf_cloud_builder.google_workflows_workflow.builder,
Step #3 - "converge-bootstrap":   on .terraform/modules/tf_cloud_builder/modules/tf_cloudbuild_builder/workflow.tf line 38, in resource "google_workflows_workflow" "builder":
Step #3 - "converge-bootstrap":   38: resource "google_workflows_workflow" "builder" {
Step #3 - "converge-bootstrap": }
Step #3 - "converge-bootstrap":     apply.go:34: 
Step #3 - "converge-bootstrap":         	Error Trace:	/builder/home/go/pkg/mod/github.com/gruntwork-io/terratest@v0.47.0/modules/terraform/apply.go:34
Step #3 - "converge-bootstrap":         	            				/builder/home/go/pkg/mod/github.com/!google!cloud!platform/cloud-foundation-toolkit/infra/blueprint-test@v0.16.1/pkg/tft/terraform.go:571
Step #3 - "converge-bootstrap":         	            				/workspace/test/integration/bootstrap/bootstrap_test.go:125
Step #3 - "converge-bootstrap":         	            				/builder/home/go/pkg/mod/github.com/!google!cloud!platform/cloud-foundation-toolkit/infra/blueprint-test@v0.16.1/pkg/tft/terraform.go:630
Step #3 - "converge-bootstrap":         	            				/builder/home/go/pkg/mod/github.com/!google!cloud!platform/cloud-foundation-toolkit/infra/blueprint-test@v0.16.1/pkg/tft/terraform.go:669
Step #3 - "converge-bootstrap":         	            				/builder/home/go/pkg/mod/github.com/!google!cloud!platform/cloud-foundation-toolkit/infra/blueprint-test@v0.16.1/pkg/utils/stages.go:31
Step #3 - "converge-bootstrap":         	            				/builder/home/go/pkg/mod/github.com/!google!cloud!platform/cloud-foundation-toolkit/infra/blueprint-test@v0.16.1/pkg/tft/terraform.go:669
Step #3 - "converge-bootstrap":         	Error:      	Received unexpected error:
Step #3 - "converge-bootstrap":         	            	FatalError{Underlying: error while running command: exit status 1; 
Step #3 - "converge-bootstrap":         	            	Error: Error creating Workflow: googleapi: Error 400: Workflows service agent does not exist
Step #3 - "converge-bootstrap":         	            	
Step #3 - "converge-bootstrap":         	            	  with module.tf_cloud_builder.google_workflows_workflow.builder,
Step #3 - "converge-bootstrap":         	            	  on .terraform/modules/tf_cloud_builder/modules/tf_cloudbuild_builder/workflow.tf line 38, in resource "google_workflows_workflow" "builder":
Step #3 - "converge-bootstrap":         	            	  38: resource "google_workflows_workflow" "builder" {
Step #3 - "converge-bootstrap":         	            	}
Step #3 - "converge-bootstrap":         	Test:       	TestBootstrap
```